src/udiskslinuxdrive.c

```diff
@@ -40,6 +40,7 @@
 #include <mntent.h>
 
 #include <glib/gstdio.h>
+#include <libstoragemgmt/libstoragemgmt.h>
```

Comment:
Do these things have to be implemented in the daemon itself? AFAICT, they could go to the *lsm* module.